### PR TITLE
This PR fixes hanging issue found with previous script while runnin node-dc-eis.

### PR DESCRIPTION
**Pull request by uttampawar**

### CODE REVIEW

**Comment by uttampawar** [comment]
BTW, functionally only run-dc-eis.sh script has really changed.

**Comment by mhdawson** [comment]
Umm, it seems like I won't be able to land this because is says there are a number of conflicts that must be resolved.

Testing it out now, but the PR will need to be rebased before I'll be able to land.

**Comment by uttampawar** [comment]
@mhdawson Okay. Mike. I'll re-base the PR and submit again.

**Comment by mhdawson** [comment]
@uttampawar  You just need to rebase and then force push to the same branch.
I ran it on 4.x and it worked ok.  Latency was a bit higher than previous runs but maybe that was a one off.

In any case once its rebased I'll recheck and then land.

**Comment by mhdawson** [comment]
I think the problem is that you continue to use the same branch as you had for earlier PRs. You either need to use a different branch or to fully rebase.  So the conflicts are mostly because of changes that were already landed being in the PR.

I created this PR with the latest changes in this one https://github.com/nodejs/benchmarking/pull/157

**Comment by mhdawson** [comment]
@uttampawar , I'm testing out https://github.com/nodejs/benchmarking/pull/157, can you check to see if it includes all of the changes that it should ?

**Comment by uttampawar** [comment]
@mhdawson Whenever you get a chance. I've resolve all th conflicts and updated my PR.

**Comment by uttampawar** [comment]
@mhdawson I've tested my latest PR with all node releases and it's seems to be working fine, no hangups and exiting cleanly. Also, I've update PR with "footprint" json format for the graphing purpose with appropriate benchid (12 and 13). 
At this time please feel free to test this PR (my repo) and then merge with mainline when it's appropriate.

**Comment by mhdawson** [comment]
Sorry I misread your comment and landed #157. Can you rebase again and then I'll test and land.

**Comment by uttampawar** [comment]
@mhdawson That's okay since I've few indentation fixes too. I'll rebase and update my PR.

File: benchmarks/dc_eis_latency/dceisLatency.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
  "name": "node-dc-eis Latency[lower is better]" ,
  "template": "../common/common_template1.html.base",
  "units": "ms",
- "outputBase" : "dceis_latency",
```
Comment (mhdawson) [inline benchmarks/dc_eis_latency/dceisLatency.json]: This change would change the name of the charts, and the names of the urls on benchmarking.nodejs.org.  I'd suggest removing and if we do want to make this change then adding in a PR were we also update the main page that shows the charts.

File: benchmarks/dc_eis_throughput/dceisThroughput.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
  "name": "node-dc-eis Ops/s[higher is better]" ,
  "template": "../common/common_template1.html.base",
  "units": "ops/s",
- "outputBase" : "dceis_throughput",
```
Comment (mhdawson) [inline benchmarks/dc_eis_throughput/dceisThroughput.json]: same comment as above will change the name of the external graphs so the main page would also have to be updated.

File: benchmarks/dc_eis_latency/dceisLatency.json (hunk 1 above)
Comment (uttampawar) [reply benchmarks/dc_eis_latency/dceisLatency.json]: @mhdawson Existing graphs on the main page have file names as above,
https://benchmarking.nodejs.org/charts/dceis_throughput.png, and
https://benchmarking.nodejs.org/charts/dceis_latency.png

It's not clear to me what needs to be changed?

Comment (mhdawson) [reply benchmarks/dc_eis_latency/dceisLatency.json]: The PR changes the name from dceis_... to dc-eis_... adding the -

Comment (mhdawson) [reply benchmarks/dc_eis_latency/dceisLatency.json]: which would break the references in https://github.com/nodejs/benchmarking/blob/master/www/index.html

**Comment by kunalspathak** [comment]
I am not following up on this closely from last few weeks, but will it solve the problem of stability in results? E.g. When I see latency numbers, for any node version, I see they are not stable. I think it will be good to investigate the flakiness to gain confidence on numbers represented by this benchmark.

image

**Comment by uttampawar** [comment]
@kunalspathak Throughput looks stable but there is some variation in latency for sure. I've not seen this much variation. I'll monitor for this week and then take some action if needed.

**Comment by uttampawar** [comment]
@mhdawson I've reverted back those name changes as suggested.

**Comment by mhdawson** [comment]
@uttampawar, this does not land cleanly due to the intervening commits.  Can you squash the changes down to 1 commit and then I can land.

**Comment by uttampawar** [comment]
@mhdawson Do you mean to combine all 22 commits into 1 I see with this PR?

**Comment by mhdawson** [comment]
@uttampawar yes, the end result should be 1 commit.

**Comment by uttampawar** [comment]
@mhdawson I think I finally got this down to 2 commits. First one merging all into one and second one resolving conflicts with mainline master branch. Please let me know if this can work and thanks for your help.

**Comment by mhdawson** [comment]
@uttampawar I still get lots of errors when I try to land.

I think you could do the following to creat a new PR

1) Copy the 7 files changed in this PR to a safe place
2) Get a clean branch.  I think you could so that with

git checkout -b cleanupDCEIS-scripts
git reset --hard XXXXX, were XXXX is the has for a commit before any of your changes
git rebase upstream/master (after doing git add remote upstream YYY, were YYY is the link to the benchmarking repo)
copy the 7 files on top of the ones.
git add ZZZ for each of the modified files
git push origin cleanupDCEIS-scripts

and then create a new PR from that branch.

**Comment by uttampawar** [comment]
@mhdawson Sorry for the inconvenience. I'll try to get it right this time.

**Comment by uttampawar** [comment]
@mhdawson Created new pull request.

**Comment by mhdawson** [comment]
Fixed via other pull request, closing.

**Pull request closed without merging**